tests/e2e/cucumber/publicLinkToFolderJourney.feature

```diff
@@ -21,3 +21,6 @@ Feature: Create public link to folder
       | resource     | name         | role     | dateOfExpiration | password |
       | folderPublic | myPublicLink | uploader | +5 days          | 12345    |
     Then "Alice" should see 1 public link
+    When the public accesses the last created public link with the correct password "12345"
+    Then the following file should not be visible on the public link
+      | lorem.txt |
```

Comment:
One should be able to see public shared content with the correct password.

Reply:
public with role **uploader** doesn't see public shared content 
describe in https://github.com/owncloud/web/issues/6389